File: MODUL 5 PEMROGRAMAN 1/PRAK501-2310817120002-SITI RATNA DWINTA SARI.py
def max_bilangan(a1, b1, c1, d1):
    max = a1

    if b1 > max:
        max = b1

    if c1 > max:
        max = c1

    if d1 > max:
        max = d1

    return max

a, b, c, d = map(int, input().split())
hasil = max_bilangan(a, b, c, d)
print(hasil)

File: MODUL 5 PEMROGRAMAN 1/PRAK501-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>

int MaxBilangan(int a1, int b1, int c1, int d1){
    int max = a1;
if (b1 > max){
    max = b1;
    }
if (c1 > max){
    max = c1;
    }
if (d1 > max){
    max = d1;
    }
    return max;

}
int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
        printf("%d", hasil);
    return 0;
}